web-ng/src/app/services/layer/layer.service.ts

```diff
@@ -30,11 +30,23 @@ import { Form } from '../../shared/models/form/form.model';
 export class LayerService {
   constructor(private dataStoreService: DataStoreService) {}
 
+  /**
+   * Creates a new layer with a generated uuid.
+   */
   createNewLayer(): Layer {
     const layerId = this.dataStoreService.generateId();
     return new Layer(layerId, /* index */ -1);
   }
 
+  /**
+   * Creates a new field with a given type, label, required, index and multipleChoice values.
+   *
+   * @param type the type of the new field.
+   * @param label the label of the new field.
+   * @param required the required value of the new field.
+   * @param index the index of the new field.
+   * @param multipleChoice the multipleChoice value of the new field.
+   */
```

Comment:
Thanks for adding these! Suggestion for adding comments in general: it's generally preferable to not repeat what's already clear from code in the comments. The idea being the extra text should add clarity rather than additional work the reader, and that if something can be clarified directly in code, even better. For example, instead of the block, we might simply write:

```
  /**
   * Creates and returns a new field with a generated unique identifier and a single English label.
   */
```

Not that method arg JSDoc are not required here, since TypeScript already lets us declare types, and there's nothing interesting to add to the comments. Could you kindly update similarly throughout?

Reply:
Done.